lesson26/src/js/login.js

```diff
@@ -0,0 +1,134 @@
+const userName = document.getElementById("username");
+const password = document.getElementById("password");
+const loginButton = document.getElementById("js-login-button");
+const storage = localStorage;
+
+if (storage.getItem("token")) window.location.href = "./index.html";
```

Comment:
名前を抽象的にして持っている意味がないので
```suggestion
if (localStorage.getItem("token")) window.location.href = "./index.html";
```

Reply:
Thanks for the review.

You're right.
I agreed that it was a useless variable, so I removed it.
[e9c3a0f](https://github.com/sae-github/JS_lesson/pull/30/commits/e9c3a0f97c5e40d8496325322d5a41af0066189d)